R/article.R

```diff
@@ -366,11 +366,50 @@ jedm_article <- function(..., keep_tex = TRUE, citation_package = "natbib") {
 #'   from <https://www.mdpi.com/authors/latex>.
 #' @export
 #' @rdname article
```

Comment:
It is not required per-se  but it seems to be a good practice yes. This allow to document all field that could be customized in YAML. 

Doing in the template `skeleton.Rmd`  in the YAML could be an option also, but it seems clearer in the help page. This will also be shown in the pkgdown website https://pkgs.rstudio.com/rticles/reference/article.html#sage-article so it is nice.